lisa/wlgen/rta.py

```diff
@@ -402,7 +402,7 @@ def _calibrate(cls, target, res_dir):
                                  calibration="CPU{}".format(cpu),
                                  res_dir=res_dir)
 
-            with rta, target.disable_idle_states(), target.freeze_userspace():
+            with rta, target.freeze_userspace():
```

Comment:
Could we give the freezer the same treatment?

Reply:
Since we need to interact with the board between each run, I'd rather avoid upsetting the userspace. Maybe things would just work, but I'd rather avoid freezing things more than really necessary (not a very hard opinion though).